cmd/check-gh-automation/main.go

```
@@ -122,7 +122,8 @@ func main() {
 	}
 
 	repos := determineRepos(o, prowAgent, logger)
-	failing, err := checkRepos(repos, o.bots.Strings(), o.appName, o.ignore.StringSet(), appCheckMode(o.appCheckMode), o.checkBranchProtection, client, logger, pluginAgent, tideQueries, prowAgent)
+	configs, _ := config.GetAllConfigs(o.releaseRepoPath)
```

We can't ignore the error here